Source/WebCore/Modules/plugins/YouTubePluginReplacement.cpp

```diff
@@ -76,16 +76,20 @@ RenderPtr<RenderElement> YouTubePluginReplacement::createElementRenderer(HTMLPlu
     if (!m_embedShadowElement)
```

Comment:
I often go with:
```
RefPtr embedShadowElement = m_embedShadowElement;
if (!embedShadowElement)
    return nullptr;
return embedShadowElement->createElementRenderer(WTFMove(style), insertionPosition);
```

Not a huge deal here since `m_embedShadowElement` is a RefPtr and not a WeakPtr though.